src/Drupal/Commands/core/UserCommands.php

```diff
@@ -121,7 +121,11 @@ public function renderRolesCell($key, $cellData, FormatterOptions $options)
      * @param string $names A comma delimited list of user names.
      * @aliases ublk,user-block
      * @usage drush user:block user3
-     *   Block the users whose name is user3
+     *   Block the user whose name is <info>user3</info>
```

Comment:
Hi @weitzman. I have updated help text and usage examples as you did for 11.x branch.